src/components/projects/projectId/attributes/attributeId/LLMResponsePlayground.tsx

```diff
@@ -13,9 +14,14 @@ type LLMResponsePlaygroundProps = {
 
 export default function LLMResponsePlayground(props: LLMResponsePlaygroundProps) {
     const dispatch = useDispatch();
+
+    const openLLMPlayground = useCallback(() => {
+        dispatch(setModalStates(ModalEnum.LLM_PLAYGROUND, { open: true, attributeId: props.attributeId, apiKey: props.apiKey }));
```

Comment:
this doesn't have the correct values for the props since it's created only once. Either ref or part of the change array